ocrolib/common.py

```diff
@@ -677,36 +678,34 @@ def expand_args(args):
 def ocropus_find_file(fname, gz=True):
     """Search for OCRopus-related files in common OCRopus install
     directories (as well as the current directory)."""
-    if os.path.exists(fname):
-        return fname
-    if gz:
-        if os.path.exists(fname + ".gz"):
-            return fname + ".gz"
-
-    for path in _data_paths:
-        full = os.path.join(path, fname)
-        if os.path.exists(full):
-            return full
-    if gz:
-        for path in _data_paths:
-            full = os.path.join(path, fname + ".gz")
-            if os.path.exists(full):
-                return full
+    possible_prefixes = [os.curdir]
+
+    local_share_path = os.path.dirname(inspect.getfile(inspect.currentframe()))
+    local_share_path = os.path.join(local_share_path,
+                                    os.pardir, os.pardir, os.pardir, os.pardir,
+                                    "share", "ocropus")
+    if os.path.isdir(local_share_path):
+        possible_prefixes += [local_share_path]
+
+    share_path = os.path.join(sysconfig.get_config_var("datarootdir"),
```

Comment:
Yes, but I thought model installation was made optional. So this covers the case where someone has some system install _with_ models, and then they made a local install (e.g., `~/.local/lib/python`) _without_ models. Which might be what some developer would do if trying to save some room.